track_oracle/example/track_reader_example.cxx

```diff
@@ -197,18 +204,38 @@ int load_tracks( const string& track_fn, track_handle_list_type& tracks, const s
   map< field_handle_type, size_t > track_stats, frame_stats;
   size_t frame_count = 0;
 
+  vul_timer timer;
   for (size_t i=0; i<tracks.size(); ++i)
   {
     vector< field_handle_type > ts = track_oracle_core::fields_at_row( tracks[i].row );
     for (size_t t=0; t<ts.size(); ++t) ++track_stats[ ts[t] ];
 
     frame_handle_list_type frames = track_oracle_core::get_frames( tracks[i] );
     frame_count += frames.size();
+    vector< oracle_entry_handle_type > frame_handles( frames.size() );
+    for (auto j=0; j<frames.size(); ++j)
+    {
+      frame_handles[j] = frames[j].row;
+    }
+    /*
     for (size_t j=0; j<frames.size(); ++j)
     {
       vector< field_handle_type > fs = track_oracle_core::fields_at_row( frames[j].row );
       for (size_t t=0; t<fs.size(); ++t) ++frame_stats[ fs[t] ];
     }
+    */
+    for (auto fields_at_frame: track_oracle_core::fields_at_rows( frame_handles ))
+    {
+      for (auto field: fields_at_frame)
+      {
+        ++frame_stats[ field ];
+      }
+    }
+    if (timer.real() > 5 * 1000)
+    {
+      LOG_INFO( main_logger, "Computed stats on " << i << " of " << tracks.size() << "; " << frame_count << " frames" );
```

Comment:
Wondering if log message should be info level of something smaller, like debug/trace. Same message applies throughout, there are a lot of new INFO messages.

Reply:
Actually, all the "LOG_INFO( ... )" calls in the examples/ programs should probably just be regular calls to cout. Willfix in the activities branch.